StackExchange.Redis/StackExchange/Redis/IDatabase.cs

```diff
@@ -45,6 +45,44 @@ public interface IDatabase : IRedis, IDatabaseAsync
         RedisValue DebugObject(RedisKey key, CommandFlags flags = CommandFlags.None);
 
         /// <summary>
+        /// Add the specified member to the set stored at key. Specified members that are already a member of this set are ignored. If key does not exist, a new set is created before adding the specified members.
+        /// </summary>
+        /// <returns>True if the specified member was not already present in the set, else False</returns>
+        /// <remarks>http://redis.io/commands/sadd</remarks>
+        bool GeoAdd(RedisKey key, RedisValue value0, RedisValue value1, RedisValue value2, CommandFlags flags = CommandFlags.None);

```

Comment:
is there a meaning to `value0` etc here? looking at http://redis.io/commands/geoadd, this is `longitude`, `latitude`, `member` - is that right? and if so: longitude and latitude should be `double`, not `RedisValue`, no? Part of me expected this to take a `GeoEntry`, but that seems to not be the case, which confused me.